libraries/ms-lib/mslib.mk

```diff
@@ -1,19 +1,19 @@
-mslib = libraries/ms-lib
+MSLIB := libraries/ms-lib
```

Comment:
This file could probably append to a list of files that your lint and clean could operate on instead of needing to explicitly add each possible library makefile.